index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+
+let name = process.argv[2];
```

Comment:
Great job on requiring your ``greet.js`` module into this module and using ``process.argv[2]``! 👍